CGPs/cgp-0130.md

```diff
@@ -0,0 +1,75 @@
+---
+cgp: 0130
+title: Aligned Public Goods Bucket: Red Beard Ventures x CELO Accelerator
```

Comment:
The second colon needs to be removed in title for the CI to pass please @reneerrusso. Here is an example

---
cgp: 129
title: RWA-focused Accelerator with Outlier Ventures
date-created: 2024-04-03
author: 'John Goldschmidt (@Johnlikes), Aron van Ammers (@aronvanammers)'
status: DRAFT
discussions-to: https://forum.celo.org/t/draft-celo-public-goods-grant-request-update-rwa-focused-accelerator-with-outlier-ventures/
governance-proposal-id:
date-executed:
---

Reply:
title: Aligned Public Goods Bucket: (<------this one) Red Beard Ventures x CELO Accelerator